### PR TITLE
fix(index): fix rendering bug on sponsor section

### PR DESCRIPTION
**Pull request by rockleona**
## Types of changes

* **Bugfix**

## Description

Remove unnecessary line to prevent rendering side-effect on sponsor section.

## Steps to Test This Pull Request

1. checkout preview deploy

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -145,14 +145,12 @@ export default {
...
-        await store.dispatch('$getSponsorsData')
-        const sponsorsData = store.state.sponsorsData
-        return {
-            sponsorsData,
+    async asyncData({ store }) {
+        if (process.client) {
+            await store.dispatch('$getSponsorsData')
         }
     },
+    fetchOnServer: false,
```
Comment (baby230211) [inline pages/index.vue:153]: Due to asyncData is used for generating static html in previous years,
i think asyncData should not be modified

**Comment by SivanYeh** [comment]
@baby230211 How about [it](https://github.com/pycontw/pycontw-frontend/pull/513/commits/e53a3cf9ecaf72585910061038c254bc9afaa73d)?

File: pages/index.vue (hunk 1 above)
Comment (rockleona) [reply pages/index.vue]: If I remember correctly, we used to fetch data when `created`.

So I just changed it to the previous workable version, and keep `asyncData` as well if we need to archive the site

Comment (rockleona) [reply pages/index.vue]: See the changes in `page/index`

https://github.com/pycontw/pycontw-frontend/commit/0293e46cd7ac514f4664666b8e0c6cf2278e5017

**Comment by baby230211** [comment]
As we discussed in discord, we found out that the asyncData return data name is conflicted with store data name.
Then we will get empty data from asyncData.

**Review by SivanYeh — APPROVED** [review]

**Review by baby230211 — APPROVED** [review]

**Pull request merged**